database/commands.js

```diff
@@ -30,6 +30,37 @@ const setSkillCheck = ({user_id, label, checked, referrer}) =>
       }
     })
 
+const logStatusUpdate = ({user_id, status}) =>
+  knex
+    .insert({
+      occurred_at: knex.fn.now(),
+      type: 'phase_4_status_update',
+      user_id,
+      metadata: {status},
+    })
+    .into('event_logs')
+
+
+const setStatus = ({user_id, status}) =>
+  logStatusUpdate({user_id, status})
+    .then(() => {
+      const insert = knex('phase_4_status').insert({
```

Comment:
this looks good. We really need an `upsert` knex helper function. we have this complexity at least twice now